Containerfile

```diff
@@ -13,7 +13,7 @@ FROM registry.fedoraproject.org/fedora:39
 # - https://github.com/osbuild/bootc-image-builder/issues/9
 # - https://github.com/osbuild/osbuild/pull/1468
 COPY ./group_osbuild-osbuild-fedora-39.repo /etc/yum.repos.d/
-RUN dnf install -y osbuild osbuild-ostree osbuild-depsolve-dnf podman qemu-img && dnf clean all
+RUN dnf install -y osbuild osbuild-ostree osbuild-depsolve-dnf podman qemu-img qemu-user-static && dnf clean all
```

Comment:
qemu-user-static is not available in RHEL, so if we ever move to a RHEL based image this will not work.

Reply:
Hm, in the general case we need `qemu-user` because some of the osbuild stages will chroot into the target. Bootupd is installed this way today, i.e. we essentially chroot into the deployment root and just call bootupd. Is that a showstopper for this approach or should we go with it for now (given that we have limited time until the demo etc)?

Reply:
Another thing that might be an issue is the arch specific mkfs things that Colin pointed out in https://github.com/osbuild/bootc-image-builder/issues/153#issuecomment-1910453402 - we need to see what we can do here.

Reply:
Lets get this working on Fedora first and then we can worry about RHEL.